src/components/MapMenu.tsx

```diff
@@ -16,79 +16,92 @@ interface MapMenuProps {
+          </div>
+          <div className="ms-1 ms-sm-3 my-1">
+            <input
+              className='btn-check'
+              type="checkbox"
+              name="greenSpaces"
+              id='greenSpaces'
+              checked={props.visibility.greenSpaces}
+              onChange={() => props.setVisibility({ ...props.visibility, 'greenSpaces': !props.visibility['greenSpaces'] })} />
+            <label className="btn btn-outline-dark" htmlFor='greenSpaces'>Green Spaces</label>
+          </div> 
+          <div className="ms-1 ms-sm-3 my-1">
+            <input
+              className='btn-check'
+              type="checkbox"
+              name="walkways"
+              id="walkways"
+              checked={props.visibility.walkways}
+              onChange={() => props.setVisibility({ ...props.visibility, 'walkways': !props.visibility['walkways'] })} />
+            <label className="btn btn-outline-dark" htmlFor='walkways'>Walkways</label>
+          </div>
+          
+          </div>
+        </div>   
+        <div className='col-12 col-lg-6'>
+        <div className='d-flex justify-content-center py-1'>
+        <div className="btn-group btn-group-sm my-1" role="group" aria-label="Basic checkbox toggle button group">
+              <input
+                className='btn-check'
+                type="checkbox"
+                name="accessibility"
+                id="accessibility"
+                checked={props.visibility.accessibility}
+                onChange={() => props.setVisibility({ ...props.visibility, 'accessibility': !props.visibility['accessibility'] })} />
+              <label className="btn btn-outline-dark d-flex align-items-center" htmlFor='accessibility'><img width={30} src='./img/handicap.png' /><span> Accessiblity</span></label>
+              <input
+                className='btn-check'
+                type="checkbox"
+                name="shuttleStops"
+                id="shuttleStops"
+                checked={props.visibility.shuttleStops}
+                onChange={() => props.setVisibility({ ...props.visibility, 'shuttleStops': !props.visibility['shuttleStops'] })} />
+              <label className="btn btn-outline-dark d-flex align-items-center" htmlFor='shuttleStops'><img width={30} src='./img/bus.png' /><span> Shuttle</span></label>
+              {/* <input
+                className='btn-check'
+                type="checkbox"
+                name="blueLightPhones"
+                id="blueLightPhones"
+                checked={props.visibility.blueLightPhones}
+                onChange={() => props.setVisibility({ ...props.visibility, 'blueLightPhones': !props.visibility['blueLightPhones'] })} />
+              <label className="btn btn-outline-dark d-flex align-items-center" htmlFor='blueLightPhones'><img width={30} src='./img/phone.png'/><span> Emergency</span></label> */}
+              <div className="btn-group" role="group">
```

Comment:
Let's remove commented code if it doesn't absolutely need to be there.

Reply:
Alright, that's gonna be the emergency button. I'm trying to bring that out of the symbol list since roger requested. It's gonna be an individual button on navbar. However I will remove it.